<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="addGroupPartyComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ryhmän nimi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addGroupGroupNameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addGroupCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addGroupAddPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lisää</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addGroupGroupNameLineEdit</tabstop>
  <tabstop>addGroupPartyComboBox</tabstop>
  <tabstop>addGroupCancelPushButton</tabstop>
  <tabstop>addGroupAddPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
